schemas/2015-10-01/Microsoft.Media.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-10-01/microsoft.media.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Media",
+  "description": "Microsoft Media Service Resource Type",
+  "resourceDefinitions": {
+    "mediaServices": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.Media/mediaServices"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-10-01"
+          ]
+        },
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-z0-9]{3,26}$"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/storageAccounts"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "location"
+      ],
+      "description": "Microsoft.Media/mediaServices"
+    }
+  },
+  "definitions" : {
+    "storageAccounts": {
+      "properties": {
+        "storageAccounts": {

```

Comment:
OK. Will add.

From: Wenjun Wu [mailto:notifications@github.com]
Sent: Wednesday, June 22, 2016 11:43 AM
To: Azure/azure-resource-manager-schemas <azure-resource-manager-schemas@noreply.github.com>
Cc: Brian Blum <brianbl@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [Azure/azure-resource-manager-schemas] Add Azure Media Service 2015-10-01 schema (#188)


In schemas/2015-10-01/Microsoft.Media.json<https://github.com/Azure/azure-resource-manager-schemas/pull/188#discussion_r68111231>:

> +            }

> +          ]

> +        }

> +      },

> +      "required": [

> +        "type",

> +        "apiVersion",

> +        "location"

> +      ],

> +      "description": "Microsoft.Media/mediaServices"

> +    }

> +  },

> +  "definitions" : {

> +    "storageAccounts": {

> +      "properties": {

> +        "storageAccounts": {

would be good if we can support expression for the "storageAccounts" property.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-resource-manager-schemas/pull/188/files/93541a94f21f4a1b95abe1e38ca14477e4b57270#r68111231>, or mute the thread<https://github.com/notifications/unsubscribe/AEH9bJScqR4tVaxMYBDlJ_t-FnEnWPYOks5qOYI5gaJpZM4I8C67>.